frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -314,7 +314,7 @@ export default function AppLayout() {
                 </DropdownMenu>
               </Sheet>
             </header>
-            <div className="flex flex-1 flex-col gap-4 p-4 lg:gap-6 lg:p-8">
+            <div className="flex flex-1 flex-col gap-4 p-4 pt-[4.5rem] md:pt-4 lg:gap-6 lg:p-8">
```

Comment:
we try to avoid using custom units, why is pt-[4.5rem] needed?

Reply:
This adds the height of the header (`h-14` i.e `56px`) to `p-4` (`16px`) that already exists making it `72px` (`4.5 rem`)

Reply:
Added margin instead